pydis_site/apps/api/models/bot/whitelist.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from pydis_site.apps.api.models.utils import ModelReprMixin
+
+
+class Whitelist(ModelReprMixin, models.Model):
+    """Whitelisted item for a type."""
+
+    INVITE = "invite"
+    EXTENSION = "extension"
+    CHANNEL = "channel"
+    ROLE = "role"
```

Comment:
hmm, what's the reason for these constants? I don't see them used anywhere else, but if they should be, I think a separate Enum would be a better fit (e.g. `WhitelistType`)

Reply:
Removed the constants.

Reply:
Hmm, did you forget to push, or am I looking at old code here? Constants are still here.

Reply:
My Bad! I removed the use of the constants but forgot to remove them.